source/clients/android.md

```diff
@@ -1,3 +1,26 @@
 # Android Client (Safe Personal Edition)
 
+## Install Mainnet Version
+
+The Mainnet version of the Gnosis Safe Android app can be downloaded from the Google PlayStore or from the GitHub releases page.
+
+Note: The versions from GitHub and the PlayStore are differently signed, therefore it is not possible to update the PlayStore version with a version downloaded from GitHub and vise-versa. Also the GitHub version will not recive automatic updates.
+
+[View on PlayStore](https://play.google.com/store/apps/details?id=pm.gnosis.heimdall)
```

Comment:
I would move these links inline to the sentence in line 5.

Reply:
If you prefer them at the bottom, I would move them to 2 lines. They seem to be rendered in 1 which confused me 
![image](https://user-images.githubusercontent.com/1380529/60589240-53067500-9d99-11e9-9a28-f10b4234afbf.png)